rssfeed/tasks.py

```diff
@@ -0,0 +1,171 @@
+from datetime import datetime
```

Comment:
See if you can break out bits of the below into separate methods and then weave them together in the poll_feed method.